internal/crossdock/server/oneway/server.go

```diff
@@ -103,6 +114,12 @@ func useCherami() bool {
 	return os.Getenv("CHERAMI") == "enabled"
 }
 
+// useGrpc checks to see if a grpc server is expected to be
+// available
+func useGrpc() bool {
+	return os.Getenv("GRPC") == "enabled"
```

Comment:
is this envVar limited to just the oneway tests?  And, if so, should we make the envVar more explicit?  (Additionally, I'm not sure how I feel about gating this test based on whether an environment variable is set, I could see some weird "i don't see the error" cases *shrug*)